src/test/java/com/coravy/hudson/plugins/github/GithubLinkActionFactoryTest.java

```diff
@@ -0,0 +1,45 @@
+package com.coravy.hudson.plugins.github;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.empty;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import com.coravy.hudson.plugins.github.GithubLinkAction.GithubLinkActionFactory;
+
+import hudson.model.Action;
+
+public class GithubLinkActionFactoryTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    public GithubLinkActionFactory factory = new GithubLinkActionFactory();
+
+    @Test
+    public void shouldCreateGithubLinkActionForJobWithGithubProjectProperty() throws IOException {
+        WorkflowJob p = j.jenkins.createProject(WorkflowJob.class, "p");
```

Comment:
use getter for instance `getInstance()` as of accessing to internal fields is not so stable as using defined getters